Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -70,8 +68,12 @@ protected override void OnApplyTemplate()
             base.OnApplyTemplate();
 
             _detailsPresenter = (ContentPresenter)GetTemplateChild(PartDetailsPresenter);
+            _masterList = (ListViewBase)GetTemplateChild(MasterList);
```

Comment:
We do not require this to be a list. Think of windows explorer. It has a TreeView. Cannot change the functionality of changing selection with the `SelectedItem`